Code fails to compile: extraneous ')' before ';'

[This line](https://github.com/apple/swift/blame/766e6231d392e488059a6fd7b89a6f1aa6a24803/lib/Sema/TypeCheckDeclPrimary.cpp#L1037) does not compile:

```
TypeCheckDeclPrimary.cpp:1037:66: error: extraneous ')' before ';'
   ctx.Diags.diagnose(initExpr->getLoc(), diag::macro_unsupported));
                                                                  ^
```

--- a/lib/Sema/TypeCheckDeclPrimary.cpp
+++ b/lib/Sema/TypeCheckDeclPrimary.cpp
@@ -1034,7 +1034,7 @@ static bool checkExpressionMacroDefaultValueRestrictions(ParamDecl *param) {
       &ctx.Diags, SF->getExportedSourceFile(),
       initExpr->getLoc().getOpaquePointerValue());
 #else
-  ctx.Diags.diagnose(initExpr->getLoc(), diag::macro_unsupported));
+  ctx.Diags.diagnose(initExpr->getLoc(), diag::macro_unsupported);
   return false;
 #endif
 }

